### DISCUSSION TITLE
REGISTER (error) ERR unknown command

### DISCUSSION THREAD

**Discussion by rkunnamp** [microsoft/garnet#595] [Q&A]
Whenever I try to run REGISTER command, I get an unknown command error.  Could some one get custom transactions working through Client-Side Command Registration?

**Comment by rkunnamp** [comment]
Answering my own question.
If my understanding is correct, doc here https://microsoft.github.io/garnet/docs/dev/custom-commands  is outdated

REGISTER perhaps might have got changed to REGISTERCS

"ExtensionBinPaths" : "/home/app"  does not work (raises the error "Encountered an issue when deserializing config file  (Path: /app/garnet.conf): 

"ExtensionBinPaths" : ["/home/app"] was found not to raise such error. Not sure if that is correct , as I could never get it working.

"ExtensionAllowOnlySignedAssemblies" mentioned in the doc has got changed to "ExtensionAllowUnsignedAssemblies" 

Perhaps the documentation regarding custom transactions need an update .

**Reply by badrishc** [reply]
Would be great if you submit a PR for this - the doc files are in the same GitHub repo under website folder.
